Code/Framework/AzCore/Tests/Math/PlaneTests.cpp

```diff
@@ -47,7 +47,13 @@ namespace UnitTest
     TEST(MATH_Plane, TestSet)
     {
         Plane pl;
+#if AZ_DEBUG_BUILD
+        AZ_TEST_START_TRACE_SUPPRESSION;
+#endif
         pl.Set(12.0f, 13.0f, 14.0f, 15.0f);
+#if AZ_DEBUG_BUILD
+        AZ_TEST_STOP_TRACE_SUPPRESSION(1);
+#endif
```

Comment:
We can avoid the `#if AZ_DEBUG_BUILD` blocks if we used the `AZ_TEST_STOP_TRACE_SUPPRESSION_NO_COUNT` macro

Reply:
I wanted to validate the right amount is triggered because there were code paths that ended up triggering two